docs/source/conf.py

```diff
@@ -43,8 +43,15 @@
 #
 # Pull from the actual release number without imports
 d = {}
-execfile(os.path.join('..', '..', 'traitsui', '_version.py'), d)
-version = release = d['version']
+try:
+    execfile(os.path.join('..', '..', 'traitsui', '_version.py'), d)
+    version = release = d['version']
+except IOError as ioe:
+    import warnings
+    msg = '''_version.py seems to be missing,
+            which is autogenerated if the users runs
```

Comment:
a clearer message would be something like "please run 'python setup.py develop' to generate this file."